www/src/examples/Carousel/Controlled.js

```diff
@@ -1,71 +1,57 @@
-class ControlledCarousel extends React.Component {
-  constructor(props, context) {
-    super(props, context);
+function ControlledCarousel() {
+  const [state, setState] = React.useState({ index: 0, direction: null });
```

Comment:
Is it more idiomatic to have separate `index` and `direction` state variables here?